cmake/CmakeFunctionHelper.cmake

```diff
@@ -45,28 +45,28 @@ set(gemm_configuration_lists "")
-  set(gemm_configuration_4 "float" 16 "true" "false" "false" 64 2 2 4 4 1 1 "local" "tall_skinny")
-  set(gemm_configuration_5 "float" 64 "true" "true" "true" 64 2 2 8 8 1 1 "local" "tall_skinny")
-  set(gemm_configuration_6 "float" 64 "true" "true" "true" 64 4 4 8 8 1 1 "local" "tall_skinny")
-  set(gemm_configuration_7 "float" 256 "true" "true" "true" 64 4 4 16 16 1 1 "local" "tall_skinny")
-  set(gemm_configuration_8 "float" 32 "true" "true" "true" 64 2 1 8 4 1 1 "local" "tall_skinny")
-  set(gemm_configuration_9 "float" 32 "true" "true" "true" 64 2 2 8 4 1 1 "local" "tall_skinny")
+  set(gemm_configuration_0 "float" 64 "true" "false" "false" 64 4 4 8 8 1 1 "local" "standard" 4)
+  set(gemm_configuration_1 "float" 64 "false" "false" "false" 64 8 8 8 8 1 1 "local" "standard" 4)
+  set(gemm_configuration_2 "float" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard" 4)
+
+  set(gemm_configuration_3 "float" 16 "true" "false" "false" 64 1 1 4 4 1 1 "local" "tall_skinny" 4)
+  set(gemm_configuration_4 "float" 16 "true" "false" "false" 64 2 2 4 4 1 1 "local" "tall_skinny" 4)
+  set(gemm_configuration_5 "float" 64 "true" "true" "true" 64 2 2 8 8 1 1 "local" "tall_skinny" 4)
+  set(gemm_configuration_6 "float" 64 "true" "true" "true" 64 4 4 8 8 1 1 "local" "tall_skinny" 4)
+  set(gemm_configuration_7 "float" 256 "true" "true" "true" 64 4 4 16 16 1 1 "local" "tall_skinny" 4)
+  set(gemm_configuration_8 "float" 32 "true" "true" "true" 64 2 1 8 4 1 1 "local" "tall_skinny" 4)
+  set(gemm_configuration_9 "float" 32 "true" "true" "true" 64 2 2 8 4 1 1 "local" "tall_skinny" 4)
   
-  set(gemm_configuration_10 "double" 64 "true" "false" "false" 64 4 4 8 8 1 1 "local" "standard")
-  set(gemm_configuration_11 "double" 64 "true" "false" "false" 64 8 8 8 8 1 1 "local" "standard")
-  set(gemm_configuration_12 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard")
+  set(gemm_configuration_10 "double" 64 "true" "false" "false" 64 4 4 8 8 1 1 "local" "standard" 4)
+  set(gemm_configuration_11 "double" 64 "true" "false" "false" 64 8 8 8 8 1 1 "local" "standard" 4)
+  set(gemm_configuration_12 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard" 4)
 
-  set(gemm_configuration_13 "double" 16 "true" "false" "false" 64 1 1 4 4 1 1 "local" "tall_skinny")
-  set(gemm_configuration_14 "double" 16 "true" "false" "false" 64 2 2 4 4 1 1 "local" "tall_skinny")
-  set(gemm_configuration_15 "double" 64 "true" "true" "true" 64 2 2 8 8 1 1 "local" "tall_skinny")
-  set(gemm_configuration_16 "double" 64 "true" "true" "true" 64 4 4 8 8 1 1 "local" "tall_skinny")
-  set(gemm_configuration_17 "double" 32 "true" "true" "true" 64 2 1 8 4 1 1 "local" "tall_skinny")
-  set(gemm_configuration_18 "double" 32 "true" "true" "true" 64 2 2 8 4 1 1 "local" "tall_skinny")
+  set(gemm_configuration_13 "double" 16 "true" "false" "false" 64 1 1 4 4 1 1 "local" "tall_skinny" 4)
```

Comment:
is tall_Skinny vectorized as well?

Reply:
Not yet but these parameters had to be added to the gemm methods so require these values even if they are ultimately not used in the kernel. I am aiming to vectorise the gemm_partial kernel as well in the future.